ipc/identity/src/evm/persistent.rs

```diff
@@ -16,8 +16,10 @@ use std::io::{BufReader, BufWriter, ErrorKind};
 use std::path::PathBuf;
 use zeroize::Zeroize;
 
+use super::Defaultable;
+
 #[derive(Default)]
-pub struct PersistentKeyStore<T> {
+pub struct PersistentKeyStore<T: Defaultable + ToString> {
```

Comment:
maybe if we dont need `T: Defaultable + ToString` in the struct definition, so that some `Impl` do not need to include `impl<T: Clone + Eq + Hash + TryFrom<KeyInfo> + Defaultable + ToString>`

Reply:
fixed. Ended up using `Default` instead